md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/AppDeplymentQueueAPI.java

```diff
@@ -0,0 +1,226 @@
+                    tableProperties.setDescription(daoProperties.getDescription());
+                    propertiesList.add(tableProperties);
+                }}
+            processExport.setProcessList(processList);
+            processExport.setPropertiesList(propertiesList);
+            LOGGER.info("export object is "+processExport);
+            AddJson addJson=new AddJson();
+            String status=addJson.addJsonToProcessId(appDeploymentQueue.getProcessId().toString(),processExport);
+
+            LOGGER.info("status of process.json addition "+status);
+            com.wipro.ats.bdre.md.dao.jpa.AppDeploymentQueue jpaAppDeploymentQueue=new com.wipro.ats.bdre.md.dao.jpa.AppDeploymentQueue();
+            AppDeploymentQueueStatus appDeploymentQueueStatus=new AppDeploymentQueueStatus();
+            appDeploymentQueueStatus.setAppDeploymentStatusId((short) 0);
+            appDeploymentQueueStatus.setDescription("pull request created");
+            jpaAppDeploymentQueue.setAppDeploymentQueueStatus(appDeploymentQueueStatus);
+            com.wipro.ats.bdre.md.dao.jpa.Process process1=new com.wipro.ats.bdre.md.dao.jpa.Process();
+            process1.setProcessId(appDeploymentQueue.getProcessId());
+            jpaAppDeploymentQueue.setProcess(process1);
+            jpaAppDeploymentQueue.setAppDomain(appDeploymentQueue.getAppDomain());
+            jpaAppDeploymentQueue.setAppName(appDeploymentQueue.getAppName());
+            Users users=new Users();
+            users.setUsername(principal.getName());
+            jpaAppDeploymentQueue.setUsers(users);
+            Long adqId=appDeploymentQueueDAO.insert(jpaAppDeploymentQueue);
+            LOGGER.info("app deployment queue Id is "+adqId);
+             returnedAppDeploymentQueue.setProcessId(appDeploymentQueue.getProcessId());
+             returnedAppDeploymentQueue.setAppName(appDeploymentQueue.getAppName());
+             returnedAppDeploymentQueue.setAppDomain(appDeploymentQueue.getAppDomain());
+             returnedAppDeploymentQueue.setUsername(principal.getName());
+             returnedAppDeploymentQueue.setAppDeploymentQueueId(adqId);
+             returnedAppDeploymentQueue.setAppDeploymentStatusId((short) 0);
+             restWrapper = new RestWrapper(returnedAppDeploymentQueue, RestWrapper.OK);
+        } catch (MetadataException e) {
+            LOGGER.error(e);
+            restWrapper = new RestWrapper(e.getMessage(), RestWrapper.ERROR);
+        }
+        return restWrapper;
+    }
+
+
+    @RequestMapping(value = "/merge/{id}", method = RequestMethod.POST)
+    @ResponseBody
+    public
+    RestWrapper merge(@PathVariable("id") Long queueId, Principal principal) {
+        RestWrapper restWrapper = null;
+        LOGGER.info("queue id is "+queueId);
+        LOGGER.info("user name is "+principal.getName());
+        AppDeploymentQueue returnedAppDeploymentQueue=new AppDeploymentQueue();
+        ProcessExport processExport = new ProcessExport();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "processExport" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)